src/wazuh_db/wdb_upgrade.c

```diff
@@ -112,6 +113,16 @@ wdb_t * wdb_upgrade_global(wdb_t *wdb) {
         }
     }
 
+    char output[OS_MAXSTR + 1] = {0};
+
+    if (version < (int)(sizeof(UPDATES)/sizeof(char*))) {
+        if (OS_SUCCESS == wdb_global_create_backup(wdb, output)) {
+            mdebug1("Create pre-upgrade global DB snapshot %s", output+3);
```

Comment:
We should change the `Create` for `Creating` or `Created`

Reply:
Yes, thanks!